### PR TITLE
Jamstack chapter 2021

### PR DESCRIPTION
**Pull request by denar90**
Progress on https://github.com/HTTPArchive/almanac.httparchive.org/issues/2156

Staging url: https://20211129t003730-dot-webalmanac.uk.r.appspot.com/en/2021/jamstack
Home page featured quote: https://20211129t003730-dot-webalmanac.uk.r.appspot.com/en/2021/?feat=jamstack#featured-chapter

todo
- [x] improve texts
- [x] update some images

### CODE REVIEW

File: src/content/en/2021/jamstack.md
Comment (Dawntraoz) [inline src/content/en/2021/jamstack.md:265]: Typo: In this case, CVW should be CWV isn't it?

**Comment by tunetheweb** [comment]
I've staged this at: https://20211129t003730-dot-webalmanac.uk.r.appspot.com/en/2021/jamstack

You can also see what the home page featured quote looks like for this chapter here: https://20211129t003730-dot-webalmanac.uk.r.appspot.com/en/2021/?feat=jamstack#featured-chapter

Updated the first comment with these links.

File: src/content/en/2021/jamstack.md
Comment (tunetheweb) [inline src/content/en/2021/jamstack.md:57]: We avoid stacked headers like these. Every heading should have at least one sentence in it explaining what this section will cover.

There's a few instances of this so could you fix them all?

File: src/content/en/2021/jamstack.md
Comment (tunetheweb) [inline src/content/en/2021/jamstack.md:143]: Nit:

```suggestion

### Adoption by subregion
```

File: src/content/en/2021/jamstack.md
Comment (tunetheweb) [inline src/content/en/2021/jamstack.md:212]: Two headings both called Performance. Avoid this.

(also stacked heading as mentioned before)

**Review by tunetheweb — CHANGES_REQUESTED** [review]
Not read it properly yet but spotted a few things for you to action.

**Comment by rviscomi** [comment]
@denar90 this is still showing as "changes requested". Is there any other feedback left to resolve or is it good for a final check before moving on to editing?

**Comment by Dawntraoz** [comment]
> @denar90 this is still showing as "changes requested". Is there any other feedback left to resolve or is it good for a final check before moving on to editing?

@rviscomi @denar90 I guess it's because 2 suggestions I made for the SEO score and Accessibility score sections. Take a look and tell me your thoughts 🤗

**Comment by denar90** [comment]
@rviscomi @Dawntraoz I've addressed all comments.

**Comment by tunetheweb** [comment]
I'm in the middle of going through this and have a few changes, if you could hold off on any more commits to this branch for now.

**Comment by tunetheweb** [comment]
Will need you to address some points in a bit @denar90 so keep a close eye for GitHub comments in next hour or so.

**Comment by denar90** [comment]
roger that :)

File: src/content/en/2021/jamstack.md
Comment (tunetheweb) [inline src/content/en/2021/jamstack.md:87]: @denar90 my general feedback is a lot of the commentary on the charts are just repeating the stats. Have you anything more to say on what the results mean?

File: src/content/en/2021/jamstack.md
Comment (tunetheweb) [inline src/content/en/2021/jamstack.md:208]: @denar90 ?

File: src/content/en/2021/jamstack.md
Comment (tunetheweb) [inline src/content/en/2021/jamstack.md:276]: @denar90

File: src/content/en/2021/jamstack.md
Comment (tunetheweb) [inline src/content/en/2021/jamstack.md:293]: Shoulld we drop the Best Practice section @denar90 ? Is it really telling us much?

File: src/content/en/2021/jamstack.md
Comment (tunetheweb) [inline src/content/en/2021/jamstack.md:419]: @denar90

File: src/content/en/2021/jamstack.md
Comment (tunetheweb) [inline src/content/en/2021/jamstack.md:453]: @denar90

File: src/content/en/2021/jamstack.md
Comment (tunetheweb) [inline src/content/en/2021/jamstack.md:469]: @denar90

File: src/content/en/2021/jamstack.md
Comment (tunetheweb) [inline src/content/en/2021/jamstack.md:503]: @denar90

File: src/content/en/2021/jamstack.md
Comment (tunetheweb) [inline src/content/en/2021/jamstack.md:509]: @denar90

**Review by tunetheweb — COMMENTED** [review]
@denar90 I've done an initial pass over in edit.

I've dropped the CrUX distribution charts. I understand they look pretty but, after discussion with @rviscomi , I don't think that data can be accurately relied on since we don't have a weighting in CrUX so merging across the data like that is questionable.

My general feedback on the chapter is that in a lot of places were listing the data shown in the figures, but without any of your expert opinion on what that means. Is there time to add any of that?

I also updated the ranking query to split by desktop and mobile.

I've also updated the staging link with the latest site.

**Comment by tunetheweb** [comment]
Oh and feel free to commit directly to this branch again and remove my `{# TODO ... #}` pieces when they've been addressed.

File: src/content/en/2021/jamstack.md
Comment (denar90) [reply src/content/en/2021/jamstack.md]: added

File: src/content/en/2021/jamstack.md
Comment (denar90) [reply src/content/en/2021/jamstack.md]: nope, last year it had lower results...

File: src/content/en/2021/jamstack.md
Comment (denar90) [reply src/content/en/2021/jamstack.md]: added

File: src/content/en/2021/jamstack.md
Comment (denar90) [reply src/content/en/2021/jamstack.md]: dropped

File: src/content/en/2021/jamstack.md
Comment (denar90) [reply src/content/en/2021/jamstack.md]: updated

File: src/content/en/2021/jamstack.md
Comment (denar90) [reply src/content/en/2021/jamstack.md]: dropped

File: src/content/en/2021/jamstack.md
Comment (denar90) [reply src/content/en/2021/jamstack.md]: added

File: src/content/en/2021/jamstack.md
Comment (denar90) [reply src/content/en/2021/jamstack.md]: updated

File: src/content/en/2021/jamstack.md
Comment (denar90) [reply src/content/en/2021/jamstack.md]: updated

**Comment by denar90** [comment]
@tunetheweb updated places with new texts or dropped at all.

**Comment by tunetheweb** [comment]
> @tunetheweb updated places with new texts or dropped at all.

Thanks. Made some slight further tweaks. Will merge this when the tests complete and then can do a round of copy editing (FYI @shantsis if able to help?)

File: src/content/en/2021/jamstack.md
Comment (tunetheweb) [inline src/content/en/2021/jamstack.md:195]:
```suggestion
Jamstack CDN services are not just for network delivery. They provide a lot of functionality to allow developers to easy deploy and manage Jamstack sites. For example, Netlify provide easy to use functionality to deploy sites in scope of their service so developers can just update the code and the continuous deployment process is managed for them. <a hreflang="en" href="https://bejamas.io/compare/netlify-vs-vercel/">Jamstack CDNs provide many other features</a> such as _serverless functions_, _A/B testing_ etc.
```

**Pull request merged**
